interface/app/$libraryId/Explorer/QuickPreview/index.tsx

```diff
@@ -181,268 +166,253 @@ export const QuickPreview = () => {
 		));
 	});
 
-	return (
-		<Dialog.Root open={open} onOpenChange={(open) => (getQuickPreviewStore().open = open)}>
-			{transitions((styles, show) => {
-				if (!show || !item) return null;
+	if (!item) return null;
 
-				const { kind, ...itemData } = getExplorerItemData(item);
+	const { kind, ...itemData } = getExplorerItemData(item);
 
-				const name =
-					newName ||
-					`${itemData.name}${itemData.extension ? `.${itemData.extension}` : ''}`;
+	const name = newName || `${itemData.name}${itemData.extension ? `.${itemData.extension}` : ''}`;
 
-				const background = !withoutBackgroundKinds.includes(kind);
-				const icon = iconKinds.includes(kind);
+	const background = !withoutBackgroundKinds.includes(kind);
+	const icon = iconKinds.includes(kind);
 
-				return (
-					<QuickPreviewContext.Provider value={{ background }}>
-						<Dialog.Portal forceMount>
-							<AnimatedDialogOverlay
-								className={clsx(
-									'absolute inset-0 z-50',
-									isDark ? 'bg-black/80' : 'bg-black/60'
+	return (
+		<Dialog.Root open={open} onOpenChange={(open) => (getQuickPreviewStore().open = open)}>
+			<QuickPreviewContext.Provider value={{ background }}>
+				<Dialog.Portal forceMount>
+					<Dialog.Overlay
+						className={clsx(
+							'absolute inset-0 z-50',
+							'data-[state=open]:animate-in data-[state=open]:fade-in-0',
```

Comment:
I think we can use utilities from `tailwindcss-radix`, same goes further down:

```suggestion
							'radix-state-open:animate-in radix-state-open:fade-in-0',
```

Reply:
True we could, will do that